### PR TITLE
add torchfix --version

### LINKED ISSUE #13: Add `torchfix --version` to print the version

### PR DESCRIPTION
**Pull request by seemethere**
Also changes the default command to not require an argument and to default to your current working directory

Fixes #13

### CODE REVIEW

File: torchfix/__main__.py [hunk 1]
```diff
@@ -14,7 +14,8 @@ def main() -> None:
 
     parser.add_argument(
         "path",
-        nargs="+",
+        nargs="*",
```
Comment (kit1980) [inline torchfix/__main__.py:17]: The current behavior is intentional to mimic ruff's API.

Comment (seemethere) [reply torchfix/__main__.py:17]: `--version` won't work otherwise since the invocation will have to look like `torchfix --version .`

Comment (seemethere) [reply torchfix/__main__.py:17]: For context this is what this looks like if you don't change that:

```
❯ torchfix --version
usage: torchfix [-h] [--fix] [-j JOBS] [--select {ALL}] [--version] [--show-stderr] path [path ...]
torchfix: error: the following arguments are required: path
```

Comment (kit1980) [reply torchfix/__main__.py:17]: Maybe it's a bigger change, but `torchfix --version` should work, and just `torchfix` should print help.

Comment (kit1980) [reply torchfix/__main__.py:17]: I think nargs is fine, but without default?

Comment (seemethere) [reply torchfix/__main__.py:17]: Okay changed!

**Review by kit1980 — APPROVED** [review]

**Pull request merged**